lib/sentry/context.ex

```diff
@@ -20,6 +20,9 @@ defmodule Sentry.Context do
         posts = Blog.list_posts()
         render(conn, "index.html", posts: posts)
       end
+
+    It should be noted that the `set_*_context/1` functions merge with the
+    existing context rather than entirely overwriting it.
```

Comment:
The original reason I created this PR was that I wasn't sure whether the `set_*_context/1` functions overwrote all previous context for a given `:user`, `:extra`, or `:tags` context. I checked the documentation for this module and it didn't clarify. Figured I'd save the next person that wonders this a few minutes.